packages/fxa-auth-server/lib/scheme-serverJWT.js

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const { verifyJWT } = require('./serverJWT');
```

Comment:
Since there are two files prefixed with `scheme` we could pull them into a specific folder.